### PR TITLE
Created tests for SpotifyProfile

### PR DESCRIPTION
**Pull request by akachriss**
Has 78% line coverage but 100% branch coverage. Only thing that is missing is the handleAuth function but I'm going to leave that for Kendal since that's her function and I'm not really sure where that leads to, also don't want to mess up anything. 

For now, I've tested the other functions that test if they're getting rendered/called correctly. Also if they're updating the main state, loggedIn based on given values.

A4AF9F0A-E21A-42A2-B4A0-399710688C35-1024x683

### CODE REVIEW

**Review by khasek — APPROVED** [review]
Very nice, good to go once the merge conflicts are resolved. I've been looking at handleAuth all day today and am not sure what that does either, so I'll probably remove it in an upcoming PR.
dunk

**Review by cjhet02 — APPROVED** [review]
Good work bestie.

**Pull request merged**
